.pre-commit-config.yaml

```diff
@@ -0,0 +1,63 @@
+---
+repos:
+  - repo: git://github.com/Lucas-C/pre-commit-hooks
+    rev: v1.1.1
+    hooks:
+      - id: remove-tabs
+
+  - repo: git://github.com/pre-commit/pre-commit-hooks
+    rev: v2.0.0
+    hooks:
+      - id: trailing-whitespace
+      - id: check-merge-conflict
+      - id: end-of-file-fixer
+      - id: name-tests-test
+      - id: check-added-large-files
+      - id: check-byte-order-marker
+      - id: check-case-conflict
+      - id: check-docstring-first
+      - id: check-json
+      - id: check-symlinks
+      - id: detect-private-key
+      - id: check-ast
+      - id: debug-statements
+
+  - repo: git://github.com/pycqa/pydocstyle.git
+    rev: 5.1.1
+    hooks:
+      - id: pydocstyle
+        exclude: '^tests*'
+
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+      - id: check-toml
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.770
+    hooks:
+      - id: mypy
+        exclude: '^(docs|tasks|tests)|setup\.py'
+        args: [--ignore-missing-imports]
+
+  - repo: https://github.com/psf/black
+    rev: 19.10b0
+    hooks:
+      - id: black
+
+  # Enable this in repositories with python packages.
+  # - repo: https://github.com/mgedmin/check-manifest
+  #   rev: '0.39'
+  #   hooks:
+  #     - id: check-manifest
+
+  - repo: https://gitlab.com/PyCQA/flake8
+    rev: '3.7.8'
+    hooks:
+      - id: flake8
+        additional_dependencies: ['pep8-naming']
+        # Ignore all format-related checks as Black takes care of those.
```

Comment:
I wanted to add `exclude: '^tests*'` here as well, so `flake8` is not checked agains `tests` directory, but instead I went with ignoring inline on lines https://github.com/thoth-station/invectio/pull/93/files#diff-c520182137ed753e8c43f2e27dbd06c912c5367100734ff6c9f635146b0ec929R1, https://github.com/thoth-station/invectio/pull/93/files?file-filters%5B%5D=.py&file-filters%5B%5D=.yaml#diff-7eacf9ce8f12f34111bc067f2516c9c1c30b34fad589938719cfd3fd764f308bR6, https://github.com/thoth-station/invectio/pull/93/files?file-filters%5B%5D=.py&file-filters%5B%5D=.yaml#diff-b34ca95fc6e4f3df60e3146e057a4e59541ef2386d9cfbe764456981b86bbd07R49